core/tools/blueos_startup_update/blueos_startup_update.py

```diff
@@ -446,6 +446,24 @@ def run_command_is_working():
     return True
 
 
+def remove_console():
+    """
+    removes "console=serial0,115200" from cmdline.txt
+    """
+    if not cmdline_file:
+        logger.error("cmdline unknown, aborting console patch")
+        return False
+    cmdline_content = load_file(cmdline_file).strip().split(" ")
```

Comment:
why are we parsing the file differently here? Can we extract the correct way to parse  this file into a function?

Reply:
yes, I intend to do that in another pr, where I also intend to stop opening it multiple times.
plus this is virtually the same structure in update_cgroups()

Reply:
okay